### PR TITLE
Add information on Data Protection for widgets

### PR DESCRIPTION
**Pull request by CooperRS**

### CODE REVIEW

File: content/widgets.md [hunk 1]
```diff
@@ -66,3 +66,5 @@
 	* Child view controller should be fine
 
 * If you have more than one widget, use the `UIApplicationShortcutWidget` Info-Plist key to specify the widget (by bundle ID) which should be shown upon 3D-touching the homescreen app icon.
+
+* Data Protection should not be enabled for widgets. If you do, iOS may fail to create a snapshot of your widget when showing the widgets in the lock screen. In this case, Springboard will crash the next time your widget should be displayed (as the snapshot is missing).
```
Comment (shagedorn) [inline content/widgets.md]: Could you specify what that means technically? Is this a capability/info plist key, and which one?

Comment (CooperRS) [inline content/widgets.md]: Sicher das. Mach ich aber dann später zuhause, dann kann ich das genauer aufschreiben. Geht um die Entitlements Datei...

Comment (shagedorn) [inline content/widgets.md]: 👍

File: content/widgets.md [hunk 2]
```diff
@@ -66,3 +66,9 @@
 	* Child view controller should be fine
 
 * If you have more than one widget, use the `UIApplicationShortcutWidget` Info-Plist key to specify the widget (by bundle ID) which should be shown upon 3D-touching the homescreen app icon.
+
+* Data Protection should not be enabled for widgets (meaning setting key ```com.apple.developer.default-data-protection``` to value a valid value in your widgets entitlements file). If you do, iOS will fail to create a snapshot of your widget after showing widgets in the lock screen. You can see a similar log entry in the console:
```
Comment (shagedorn) [inline content/widgets.md]: (do not set `com.apple.developer.default-data-protection` to a valid value in your widget's entitlements file).
(use single ` (not three of them) for inline code format)

If you do, iOS will fail to create a snapshot of your widget on the lock screen. You will see a log entry in the console similar to this:

File: content/widgets.md [hunk 3]
```diff
@@ -66,3 +66,9 @@
...
+> ImageIO: IIOImageWriteSession:111: cannot create: '/private/var/mobile/Containers/Data/PluginKitPlugin/C4C7DD84-E4FC-4507-8D77-364970936BD8/Library/Caches/com.apple.notificationcenter/Snapshots/de.cooperrs.Authenticator.Authenticator-Widget-PhoneTall-359-ksh-Latn_DE-UICTContentSizeCategoryL-NCWidgetDisplayModeCompact.ca/assets/image0.ktx.sb-e8f991c7-ZpFG9m'
```
Comment (shagedorn) [inline content/widgets.md]: You probably want to remvove "personal" details:

> ImageIO: IIOImageWriteSession:111: cannot create: > /private/var/mobile/Containers/Data/PluginKitPlugin/C4C7DD84-E4FC-4507-8D77-364970936BD8/Library/Caches/com.apple.notificationcenter/Snapshots/...-UICTContentSizeCategoryL-NCWidgetDisplayModeCompact.ca/assets/image0.ktx.sb-e8f991c7-ZpFG9m'

File: content/widgets.md [hunk 4]
```diff
@@ -66,3 +66,9 @@
...
+> error = 1 (Operation not permitted)
+	 
+	 In this case, Springboard will crash the next time your widget should be shown (as the snapshot is missing).
```
Comment (shagedorn) [inline content/widgets.md]: Remove leading tab

**Pull request merged**